pig/src/main/java/org/jboss/pnc/bacon/pig/impl/pnc/PncBuilder.java

```diff
@@ -112,9 +115,12 @@ boolean isSuccessfullyFinished(String groupBuildId) {
                 case SUCCESS:
                     return verifyAllBuildsInGroupBuildInFinalStateWithProperCount(groupBuildId);
                 default:
-                    throw new RuntimeException("Build group failed " + groupBuild);
+                    throw new RuntimeException(
+                            "Build group failed " + new URL(
```

Comment:
Why use `new URL`? Also isn't it `group-builds` not `group-build`? 
Why not make it 
```
throw new FatalException("Build group failed http://{}/pnc-web/#/group-builds/{}",
		 Config.instance().getActiveProfile().getPnc().getUrl(), groupBuildId)
```

Reply:
Made it url so it's clickable but it's working without it too so you're right. For formatting I think it's more readable my way but I don't mind changing in, will send fix.